.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Note**: Please only use this template to file bugs with the `chatkit-client-js` SDK. For general queries dedicated support is always available at https://support.pusher.com and support@pusher.com
```

Comment:
```suggestion
**Note**: Please only use this template to file bugs with the `chatkit-client-js` SDK. For general queries dedicated support is always available at [support.pusher.com](https://support.pusher.com) and [support@pusher.com](mailto:support@pusher.com)
```

?

But then again, this is only ever going to be seen by people in plaintext, so maybe the markup is more of a distraction than it's worth... feel free to ignore.